.travis.yml

```diff
@@ -353,11 +412,12 @@ matrix:
             - ubuntu-toolchain-r-test
 
     - os: linux
-      env: TOOLSET=gcc COMPILER=g++-6 CXXSTD=c++14 TEST_SUITE="compile_fail examples"
+      env: TOOLSET=gcc COMPILER=g++-6 CXXSTD=c++14 TEST_SUITE="examples"
```

Comment:
What happened to the compile_fail test suite in this row?

Reply:
You are right. That is something, what should be there.